src/legacy/core_plugins/data/public/search/index.ts

```diff
@@ -20,3 +20,5 @@
 export * from './aggs';
 export { getRequestInspectorStats, getResponseInspectorStats } from './utils';
 export { serializeAggConfig } from './expressions/utils';
+// @ts-ignore
+export { tabifyAggResponse, tabifyGetColumns } from '../../common/tabify';
```

Comment:
Why do we need to re-export this here?
Any consumer within the plugin can import from `common` and externally, it will be imported from the `index.ts` file